### ISSUE TITLE
Describe the data model in plain English

### ISSUE THREAD

**Issue by pwalsh** [openbudgets/data-model#1]
Hi @jakubklimek 

I see the data model is in Turtle. 

In my opinion, this unnecessarily obfuscates its understanding, and also inextricably ties this understanding to a specific implementation of the model.

As I see it, we have two important meta goals that are met with a plain English description, or at least an RFC-style spec:
1. All stakeholders, technically inclined and otherwise, can understand the proposal.
2. The ability to work from a more textual representation towards actual implementations - this is critical also to work towards a fully working OpenSpending Data Package of the same data.

What are your thoughts?

**Comment by jakubklimek** [comment]
@pwalsh D1.2 and D1.3 are "demonstrator" deliverables containing the RDF DCV DSDs themselves and a short description, which of course will be in plain English and is being developed here (right now it only captures our thoughts on the model): https://docs.google.com/document/d/1MIGpwDG3MDizrhDIn9jZ6-bP_mmMXFUAgiovIf-21cI

A more detailed documentation will be developed as part of D1.4 and finalised in M18 in D1.5.

**Comment by jindrichmynarz** [comment]
@pwalsh: Can we close this by linking to [D1.4](https://docs.google.com/document/d/1QvTj1k1Ifln47lQzBdXzC5heZYaxbYVxoKWqMsK7qNU/edit)?

**Comment by pwalsh** [comment]
oh yeah!
